File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/forecast/Temperature.java
package com.homecentral.jrs.hjemmesentral.model.yr.forecast;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

@Root(name = "temperature", strict = false)
public class Temperature {

    @Attribute(name="unit", required = false)
    String unit;

    @Attribute(name="value", required = false)
    String value;

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/forecast/Temperature.kt
package com.homecentral.jrs.hjemmesentral.model.yr.forecast

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root

@Root(name = "temperature", strict = false)
data class Temperature @JvmOverloads constructor(
        @field:Attribute(name = "value") var value: String = "",
        @field:Attribute(name = "unit") var unit: String = ""
)
